src/test/java/net/snowflake/client/jdbc/ResultSetMultiTimeZoneLatestIT.java

```diff
@@ -256,6 +256,32 @@ private void helperTestUseSessionTimeZone(
     assertEquals(true, rs.getTime("COLD").toString().equals(expectedTime2));
     // Cannot getTime() for Date column (colE)
 
+    // Test special case for timestamp_tz (offset added)
+    // create table with of type timestamp_tz
+    statement.execute("create or replace table tabletz (colA timestamp_tz)");
+    prepSt = connection.prepareStatement("insert into tabletz values(?), (?)");
+    // insert 2 timestamp values, but add an offset of a few hours on the end of each value
+    prepSt.setString(
+        1, expectedTimestamp + " +0500"); // inserted value is 2019-01-01 17:17:17.6 +0500
+    prepSt.setString(
+        2, expectedTimestamp2 + " -0200"); // inserted value is 1943-12-31 01:01:33.0 -0200
+    prepSt.execute();
+
+    rs = statement.executeQuery("select * from tabletz");
+    rs.next();
+    // Assert timestamp is displayed with no offset when flag is true. Timestamp should look
+    // identical to inserted value
+    assertEquals(useSessionTimezone, rs.getTimestamp("COLA").toString().equals(expectedTimestamp));
```

Comment:
why not do something like
```
if (useSessionTimeZone) {
  assertEquals(expectedTimestamp, rs.getTimestamp("COLA").toString());
  ....
}
```

Reply:
Ah, I just did the current format to cut down on repeated code. I run this same helper test function with useSessionTimeZone=true, useSessionTimeZone=false, and then both of those with a combination of other formatting session parameters to make sure there's no unexpected behavior with multiple session params enabled.

Reply:
I know but it seems hard to read. If you can separate the assertions for two cases then I believe it can improve the readability.

Reply:
Changed!